Vungle/src/main/java/com/mopub/mobileads/VungleRouter.java

```diff
@@ -371,6 +379,13 @@ public VungleSettings applyVungleNetworkSettings(Map<String, String> configurati
         if (configuration == null || configuration.isEmpty()) {
             return VungleNetworkSettings.getVungleSettings();
         }
+        try {
+            boolean optedIn = Boolean.parseBoolean(configuration.get("VNG_CCPA_CONSENT_STATUS"));
```

Comment:
don't we need the same for ConsentStatus?

Reply:
For GDPR, MoPub provides PersonManager framework, we no need to handle it in configuration.